ci-operator/step-registry/redhat-appstudio/conformance-tests/redhat-appstudio-conformance-tests-commands.sh

```
@@ -0,0 +1,78 @@
+if [[ -n "${GINKGO_LABEL_FILTER:-}" ]]; then
+    LABEL_ARGS="-ginkgo.label-filter=${GINKGO_LABEL_FILTER}"
+fi
+
+# Run conformance tests
+echo "Running conformance tests from test/go-tests..."
+cd test/go-tests
+go test -mod=mod ./tests/conformance \
+    -v \
+    -timeout "${GINKGO_TEST_TIMEOUT:-30m}" \
+    -ginkgo.vv \
+    ${LABEL_ARGS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)